deploy/binary-build-and-deploy-scripts/install-driver.sh

```diff
@@ -4,8 +4,7 @@ set -ex
 DRIVER_LOCATION="/host/usr/libexec/kubernetes/kubelet-plugins/volume/exec/ibm~ibmc-s3fs"
 KUBELET_SVC_CONFIG="/host/lib/systemd/system/kubelet.service"
 
-mkdir -p /host/local/bin
-cp /root/bin/s3fs /host/local/bin/
+cp /root/bin/s3fs /host/usr/local/bin/
 cp /root/bin/install-dep.sh /host/root/
 chmod +x /host/local/bin/s3fs /host/root/install-dep.sh 
```

Comment:
Need to also change this path to `/host/usr/local/bin/s3fs`

Reply:
@fplk i mean for line 10, you are calling `chmod` on a missing path

Reply:
Oh, I see - sorry, I was looking at the wrong spot. Sure, good point. It worked for me without, but you're right we should definitely change that. Thank you.